app/src/main/java/com/github/pockethub/ui/MainActivity.java

```diff
@@ -86,7 +87,11 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        Bugsnag.init(this);
+
+        if (!BuildConfig.DEBUG) {
+            Bugsnag.init(this);
```

Comment:
According to the docs, this should be called in the Application, not in one Activity:

> In your application's onCreate function, initialize Bugsnag to begin capturing exceptions:
```java
Bugsnag.init(this);
```

https://bugsnag.com/docs/notifiers/android